### PR TITLE
update all components then  responsive done

### PR DESCRIPTION
**Pull request by chandrahmani**
Aligned product listing to the single row and showing 4 items

https://github.com/chandrahmani/hello-travel/issues/3

Add react router-dom v6 

https://github.com/chandrahmani/hello-travel/blob/future-f1/src/App.jsx

## Screenshots
Products listing
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
     integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
-  <title>Hello Treval</title>
+  <title> Hello Treval </title>
```
Comment (hidaytrahman) [inline index.html:10]:
```suggestion
  <title>Hello Travel India </title>
```

Comment (hidaytrahman) [reply index.html:10]: Spelling mistake

File: src/components/adventur/Advent.css [hunk 2]
```diff
@@ -16,11 +16,15 @@ footer {
...
+    font-size: 25px;
+    height: 15vh;
     padding: 1rem;
 }
 
+.text-wrp {
+    margin-left: 1rem;
+}
+
 .bttn {
```
Comment (hidaytrahman) [inline src/components/adventur/Advent.css:28]: it should be `btn` not `bttn`

File: src/components/adventur/Adventur.jsx [hunk 3]
```diff
@@ -33,9 +33,11 @@ const Adventor = () => {
             <footer>
                 <h1 className='txts'>ustomize Your Holiday Package With Our Travel Experts</h1>
                 <div className='bttn-wrp'>
-                    <button className='bttn'>
-                        Get customize Quotes
-                    </button>
+                    <div className='bttn-wrp'>
+                        <div className='web-link'>
+                            <a href='https://www.hellotravelindia.in/'>webtechpie.com</a>
```
Comment (hidaytrahman) [inline src/components/adventur/Adventur.jsx:38]: Put webtechpie.com link

**Review by hidaytrahman — COMMENTED** [review]
Minor adjustment needed

**Comment by hidaytrahman** [comment]
@chandrahmani -  Whats is the update?

File: index.html (hunk 1 above)
Comment (hidaytrahman) [reply index.html:10]: its not solved

File: src/components/adventur/Adventur.jsx (hunk 3 above)
Comment (chandrahmani) [reply src/components/adventur/Adventur.jsx]: update link add webtechpie.com link

File: src/components/adventur/Advent.css (hunk 2 above)
Comment (chandrahmani) [reply src/components/adventur/Advent.css]: --update className btn

Comment (hidaytrahman) [reply src/components/adventur/Advent.css]: you don't need to confirm if you have done the changes

File: package.json [hunk 4]
```diff
@@ -11,16 +11,21 @@
     "deploy": "gh-pages -d dist"
   },
   "dependencies": {
+    "@phosphor-icons/core": "^1.4.5",
```
Comment (hidaytrahman) [inline package.json:14]: why you have added `@phosphor-icons/core`

File: src/App.jsx [hunk 5]
```diff
@@ -1,11 +1,27 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import './App.css'
+import Adventur from './components/adventur/Adventur';
```
Comment (hidaytrahman) [inline src/App.jsx:3]: wrong spelling `Adventur`. for all of this words

File: src/App.jsx [hunk 6]
```diff
@@ -1,11 +1,27 @@
...
+import Navbar from './components/core/navbar/Navbar';
 import Header from './components/header/Header'
+import Servic from './components/service/Servic';
```
Comment (hidaytrahman) [inline src/App.jsx:6]: wrong spell `Servic`

 use this `Service`

File: src/App.jsx [hunk 7]
```diff
@@ -1,11 +1,27 @@
...
-      <Header />
+
+      <BrowserRouter>
+        <Navbar />
+        <Header />
+        <Routes>
+          <Route path='/' element={<Header />}>
+            <Route path='' element={<Servic />} />
+            <Route path='adventur' element={<Adventur />} />
+            <Route path='travel' element={<TravelDtl />} />
```
Comment (hidaytrahman) [inline src/App.jsx:21]: TravelDetails

File: src/components/adventur/Advent.css [hunk 8]
```diff
@@ -1,6 +1,12 @@
+.advent-section {
```
Comment (hidaytrahman) [inline src/components/adventur/Advent.css:1]: ????? correct it

File: src/components/adventur/Advent.css [hunk 9]
```diff
@@ -1,6 +1,12 @@
...
+}
+
 .advent-container {
     display: flex;
-    padding: 4rem;
+    padding: 3rem;
+    margin-left: 3rem;
 }
 
 .anchor {
```
Comment (hidaytrahman) [inline src/components/adventur/Advent.css:12]: link or a

File: src/components/adventur/Advent.css [hunk 10]
```diff
@@ -11,17 +17,23 @@
 
 footer {
     background-color: #5d3d90 !important;
+    display: flex;
+    justify-content: center;
 }
 
 .txts {
```
Comment (hidaytrahman) [inline src/components/adventur/Advent.css:24]: what it does, show me the screenshot?

File: src/components/adventur/Advent.css [hunk 11]
```diff
@@ -11,17 +17,23 @@
...
     text-align: center;
-    font-size: 30px;
-    height: 20vh;
+    font-size: 25px;
+    height: 8vh;
     padding: 1rem;
 }
 
-.bttn {
+.text-wrp {
```
Comment (hidaytrahman) [inline src/components/adventur/Advent.css:32]: wrp? wrapper

File: src/components/adventur/Advent.css [hunk 12]
```diff
@@ -31,19 +43,78 @@ footer {
...
+    }
+}
 
+@media screen and (max-width:768px) {
+    .advent-container {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .num-wrp {
```
Comment (hidaytrahman) [inline src/components/adventur/Advent.css:83]: num ???

File: src/components/adventur/Adventur.jsx [hunk 13]
```diff
@@ -1,45 +1,59 @@
...
+                            </div>
 
-                        <div >
                             <div >
-                                <p c>Welcome to your 'Glorious Kashmir' tour! Arrive into Airport in Srinagar. You will be met and transferred to your houseboat in Srinagar. When we think of Kashmir, the first thing that comes to our Mind is 'Houseboats'. Staying on a Houseboat means like living in Paradise, an experience one is not likely to find anywhere else on the surface of the earth. In the evening, enjoy a romantic Shikara ride for one hour on the Dal lake. Overnight in Houseboat.</p>
+                                <div >
+                                    <p c>Welcome to your 'Glorious Kashmir' tour! Arrive into Airport in Srinagar. You will be met and transferred to your houseboat in Srinagar. When we think of Kashmir, the first thing that comes to our Mind is 'Houseboats'. Staying on a Houseboat means like living in Paradise, an experience one is not likely to find anywhere else on the surface of the earth. In the evening, enjoy a romantic Shikara ride for one hour on the Dal lake. Overnight in Houseboat.</p>
```
Comment (hidaytrahman) [inline src/components/adventur/Adventur.jsx:31]: what is c?

File: src/components/adventur/Adventur.jsx [hunk 14]
```diff
@@ -1,45 +1,59 @@
...
+                <Container>
+                    <h1 className='txts'>ustomize Your Holiday Package With Our Travel Experts</h1>
+                    <div className='link-wrp'>
+                        <div className='link-container'>
+                            <div className='web-link'>
+                                <a href='https://www.webtechpie.com/'>webtechpie.com</a>
+                            </div>
+                        </div>
+                        <div className='num-wrp'>
+                            <a href='#' ht className='num'>
```
Comment (hidaytrahman) [inline src/components/adventur/Adventur.jsx:48]: ??? ht

File: src/components/core/navbar/Button.jsx [hunk 15]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Button.css";
+
+
+const Style = ['btn--primary', 'btn--outline']
+
+const Size = ['btn--medium', 'btn--large']
```
Comment (hidaytrahman) [inline src/components/core/navbar/Button.jsx:8]: why  this is an array?

File: src/components/core/navbar/Button.jsx [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+
+export const Button = ({
+    children,
+    type,
+    onclick,
+    buttonStyle,
+    buttonSzie
+}) => {
+    const checkButtonStylev = Style.includes(buttonStyle)
```
Comment (hidaytrahman) [inline src/components/core/navbar/Button.jsx:17]: what is `checkButtonStylev` ?

File: src/components/core/navbar/Navbar.jsx [hunk 17]
```diff
@@ -0,0 +1,70 @@
+import { List, X } from "phosphor-react";
+import { useState } from "react";
+import { Link } from "react-router-dom"
+import { Button } from "./Button";
+import './Navbar.css'
+// import img from './travel.png'
```
Comment (hidaytrahman) [inline src/components/core/navbar/Navbar.jsx:6]: 🤖

File: src/components/core/navbar/Navbar.jsx [hunk 18]
```diff
@@ -0,0 +1,70 @@
...
+
+    const showButton = () => {
+        if (window.innerWidth <= 960) {
+            setButton(false)
+        } else {
+            setButton(true)
+        }
+    }
+
+    return (<>
```
Comment (hidaytrahman) [inline src/components/core/navbar/Navbar.jsx:29]: indent or formatting

File: src/components/core/navbar/Navbar.jsx [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+                    </li>
+
+                    <li className="nav-item">
+                        <Link to="/services" className="nav-links" onClick={closeMobileMenu}>
+                            Services
+                        </Link>
+                    </li>
+
+                    <li className="nav-item">
+                        <Link to="/adventur" className="nav-links" onClick={closeMobileMenu}>
```
Comment (hidaytrahman) [inline src/components/core/navbar/Navbar.jsx:53]: ??

File: src/components/header/Header.css [hunk 20]
```diff
@@ -103,26 +107,101 @@
     font-weight: 900;
 }
 
-@media screen and (max-width:900px) {
+.phn-num {
```
Comment (hidaytrahman) [inline src/components/header/Header.css:110]: .phn?

File: src/components/header/Header.css [hunk 21]
```diff
@@ -103,26 +107,101 @@
...
+    left: 128px;
+    display: inline-block;
+    padding: 0.4rem;
+    font-size: 20px;
+    margin-left: 1rem;
+    top: 109px;
+    margin-top: 1rem;
+}
+
+.header-flex {
```
Comment (hidaytrahman) [inline src/components/header/Header.css:123]: why header flex?

File: src/components/header/Header.jsx [hunk 22]
```diff
@@ -1,86 +1,113 @@
...
 import "./Header.css"
+import { Container } from "./Styled";
 const Header = () => {
 
     const { register, handleSubmit, formState: { errors } } = useForm();
 
-    const onSubmit = () => {
-
+    const onSubmit = (data) => {
+        console.log(data)
```
Comment (hidaytrahman) [inline src/components/header/Header.jsx:11]: 🤖

File: src/components/service/Servic.jsx [hunk 23]
```diff
@@ -57,26 +58,28 @@ const Servic = () => {
...
 
             <section className='section-tow'>
-                <div className='section-conatiner' >
-                    <p>
-                        <span className='txt'>About Kashmir Tourism</span>< br />
-                        "Paradise on the Earth" the state is world famous for its scenic splendour, snow-capped mountains, plentiful wildlife, hospitable people and local handicraft.
-                        With apple orchards, saffron fields, glittering lakes & breathtaking meadows, Kashmir, in the union territory of Jammu & Kashmir, draws the attention of tourists from across the globe. With tranquil settings, magnificent views, and pristine nature, Kashmir is a perfect holiday destination that should be visited once in a lifetime.
-                        While the beautiful Dal and Nigeen lakes in Srinagar offer an opportunity to relish a stay at the houseboat and enjoy a Shikara Ride; the alpine lakes like Tarsar Marsar, Vishnasar, Gadsar Lakes call out the adventure lovers for the best trekking experience.</p>
+                <Container >
+                    <div className='section-conatiner' >
```
Comment (hidaytrahman) [inline src/components/service/Servic.jsx:68]: conatiner??

File: src/components/service/Servic.jsx [hunk 24]
```diff
@@ -57,26 +58,28 @@ const Servic = () => {
...
+                        <p className='text-wrapper'>
+                            <span className='txt'>About Kashmir Tourism</span>< br />
+                            "Paradise on the Earth" the state is world famous for its scenic splendour, snow-capped mountains, plentiful wildlife, hospitable people and local handicraft.
+                            With apple orchards, saffron fields, glittering lakes & breathtaking meadows, Kashmir, in the union territory of Jammu & Kashmir, draws the attention of tourists from across the globe. With tranquil settings, magnificent views, and pristine nature, Kashmir is a perfect holiday destination that should be visited once in a lifetime.
+                            While the beautiful Dal and Nigeen lakes in Srinagar offer an opportunity to relish a stay at the houseboat and enjoy a Shikara Ride; the alpine lakes like Tarsar Marsar, Vishnasar, Gadsar Lakes call out the adventure lovers for the best trekking experience.</p>
+
+                        <div className='imgs'>
```
Comment (hidaytrahman) [inline src/components/service/Servic.jsx:75]: imgg ?

File: src/components/travelDtl/TravelDtl.jsx [hunk 25]
```diff
@@ -25,24 +26,30 @@ const TravelDtl = () => {
...
-                                <h4>{item.package}</h4>
-                                <h5>Rs-/ {item.price}</h5>
-                                <p>
-                                    {item.place}
-                                </p>
-                                <button className="btns">Get Quote</button>
+                <Container>
+                    <div className="card-flex">
+                        {data.map((item, index) =>
+                            <div className="grid-container" key={index}>
```
Comment (hidaytrahman) [inline src/components/travelDtl/TravelDtl.jsx:32]: key  will always have the unique id not the index if we have option to add ID

File: src/components/travelDtl/TravelDtl.jsx [hunk 26]
```diff
@@ -25,24 +26,30 @@ const TravelDtl = () => {
...
+                                <div className="grid-item">
+                                    <div className="info-wrappers">
```
Comment (hidaytrahman) [inline src/components/travelDtl/TravelDtl.jsx:34]: singular?

**Review by hidaytrahman — CHANGES_REQUESTED** [review]
Try to resolve all the comments

File: src/App.jsx (hunk 5 above)
Comment (chandrahmani) [reply src/App.jsx:3]: 👍

File: src/App.jsx (hunk 6 above)
Comment (chandrahmani) [reply src/App.jsx:6]: 👍

File: src/components/adventur/Adventur.jsx (hunk 13 above)
Comment (chandrahmani) [reply src/components/adventur/Adventur.jsx]: added by mistake

File: src/components/adventur/Adventur.jsx (hunk 14 above)
Comment (chandrahmani) [reply src/components/adventur/Adventur.jsx]: added by mistake

File: src/components/core/navbar/Navbar.jsx (hunk 17 above)
Comment (chandrahmani) [reply src/components/core/navbar/Navbar.jsx:6]: 👍

File: src/components/adventur/Advencher.css [hunk 27]
```diff
@@ -0,0 +1,118 @@
+.advencher-section {
+    display: flex;
+    justify-content: center;
+}
+
+.advencher-container {
```
Comment (hidaytrahman) [inline src/components/adventur/Advencher.css:6]: adventure

File: src/components/header/Header.css [hunk 28]
```diff
@@ -103,26 +110,95 @@
...
+        left: 0px;
+        color: #8492a6 !important;
+        font-size: 18px;
+    }
+
+    .img-align img {
+        border-radius: 7px;
+    }
+
+    .info-conatiner {
```
Comment (hidaytrahman) [inline src/components/header/Header.css:189]: ??? spelling

File: src/components/travelDtl/TravelDetails.jsx [hunk 29]
```diff
@@ -0,0 +1,58 @@
...
+import { Container } from "../header/Styled";
+import Service from "../service/Service";
+
+const TravelDetails = () => {
+    const [data, setData] = useState([])
+
+    const travelData = async () => {
+        try {
+            const { data } = await axios.get('data.json')
+            console.log(data)
```
Comment (hidaytrahman) [inline src/components/travelDtl/TravelDetails.jsx:13]: 🤖

File: src/components/travelDtl/TravelDetails.jsx [hunk 30]
```diff
@@ -0,0 +1,58 @@
...
+                                        <span className="offer">{item.offer}</span>
+                                        <img src={item.url} />
+                                        <span className="offers">{item.day}</span>
+                                        <h3>{item.name}</h3>
+                                        <h4>{item.package}</h4>
+                                        <h5>Rs-/ {item.price}</h5>
+                                        <p>
+                                            {item.place}
+                                        </p>
+                                        <button className="btns">Get Quote</button>
```
Comment (hidaytrahman) [inline src/components/travelDtl/TravelDetails.jsx:44]: btns?

**Review by hidaytrahman — CHANGES_REQUESTED** [review]

File: src/components/service/Servic.jsx (hunk 23 above)
Comment (chandrahmani) [reply src/components/service/Servic.jsx]: 👍

**Review by hidaytrahman — APPROVED** [review]

**Pull request merged**